zeppelin-server/src/main/java/org/apache/zeppelin/rest/SecurityRestApi.java

```diff
@@ -109,6 +109,8 @@ public Response ticket() {
   @GET
   @Path("userlist/{searchText}")
   public Response getUserList(@PathParam("searchText") final String searchText) {
+
+    final int numUsersToFetch = 5;
```

Comment:
Can you use this same variable for line no 164 (`if (maxLength == 5) {`) as well.

Reply:
@prabhjyotsingh Done the changes.